src/model/regions_info.js

```diff
@@ -294,6 +294,25 @@ export default class RegionsInfo extends EventSubscriber {
                             shapes.set(shapeId, newShape);
                             count++;
                         });
+                        //sort the shapes map
```

Comment:
I think we can actually do it without the tmp array. sort should be doing it in place so that you could call the sort with comparator before the array.map line further up and that should do the trick.